superset/connectors/druid/models.py

```diff
@@ -520,6 +528,9 @@ def get_perm(self):
             '[{obj.cluster_name}].[{obj.datasource_name}]'
             '(id:{obj.id})').format(obj=self)
 
+    def update_from_object(self, obj):
```

Comment:
@mistercrunch is there any reason why this wasn't/hasn't been implemented for Druid datasources? The UX is fairly poor if people try to update the metadata fro a Druid datasource which seems to be successfully saved yet the datasource hasn't been updated.